modules/riemann/riemann-grammar.ym

```diff
@@ -90,7 +93,7 @@ riemann_option
           {
             riemann_dd_set_field_service(last_driver, $3);
           }
-        | KW_EVENT_TIME '(' template_content ')'
+        | KW_EVENT_TIME '(' template_content event_time_unit ')'
```

Comment:
Do we have a reason not to save the time with ms accuracy if possible ?

Reply:
We spoke IRL about this.
Before this PR, the value provided in the `event_time` is a number which should be in seconds granularity. Because in this field a number is provided without any dimension, and it shall be determine the precision.